test/docker_container_test_case.py

```diff
@@ -100,5 +101,9 @@ def _kill_containers(cls):
 
     @classmethod
     def tearDownClass(cls):
+        for containers in cls._containers:
+            for line in containers.logs().decode().split('\n'):
+                if 'Deprecated' in line:
+                    warnings.warn(line, DeprecationWarning)
```

Comment:
What would such a line of the container output be, for example?

Reply:
And isn't this useful in general? If it is, this change shouldn't be dropped.

Reply:
The line within the container logs looks like:
`'[2021-01-28T01:49:24,844][WARN ][o.e.d.c.ParseField       ] [9Bhi2zf] Deprecated field [exclude] used, expected [excludes] instead')`

The AssertionError in python is more verbose:
`AssertionError: ["{message : DeprecationWarning('[2021-01-28T01:49:24,844][WARN ][o.e.d.c.ParseField       ] [9Bhi2zf] Deprecated field [exclude] used, expected [excludes] instead'), category : 'DeprecationWarning', filename : '/home/travis/build/DataBiosphere/azul/test/docker_container_test_case.py', lineno : 107, line : None}"]`

Yes this feature of extracting deprecation warnings out of containers that we use is useful; additional deprecation warnings would only occur if we change the versioning of Elasticsearch, or modifications to the requirements, so it would be nice to see these warnings. 

If we update to ES7.0, the Elasticsearch-py module will raise these errors so they would be more visible in the Azul logs. https://elasticsearch-py.readthedocs.io/en/7.10.0/Changelog.html#id25

If we switch over to a long-living Elasticsearch instance for unit-tests (#2653) we should try to retain this feature.

Reply:
Remind me to discuss at parking lot. (Note to self: reorder commits, and add a drop commit so the versioning deprecation can be observed before we switch to the new versioning)

Reply:
From standup discussion: 
1.  Add container changes (see all deprecation warnings)
2. Eliminate "other" deprecation warnings ('excludes'...) (see only version deprecation warning)
3. Add fixes for version deprecation warnings